src/ServerSettings.cpp

```diff
@@ -70,7 +71,35 @@ ServerSettings::identifyServerSetting(const std::string &token)
 	// TODO: this might need a costum exception
 }
 
-// TODO: void addLocationSetting(LocationSettings settings);
+const std::string
+methodToString(HTTPMethod method) // TODO: change data_types in function
```

Comment:
move to HTTPMethod.cpp file

Reply:
Nowhere to move to. HTTPMethod is not a class but a method